modules/installation-mirror-repository.adoc

```diff
@@ -38,9 +38,10 @@ $ export LOCAL_REPOSITORY='<repository_name>' <3>
 $ export PRODUCT_REPO='openshift-release-dev' <4>
 $ export LOCAL_SECRET_JSON='<path_to_pull_secret>' <5>
 $ export RELEASE_NAME="ocp-release" <6>
+$ export ARCHITECTURE=<server_architecture> <7>
```

Comment:
@jianlinliu, since this variable's split for upgrade, does it also need to be split for install?

Reply:
To keep consistent, yeah, it also can be split for install.